packages/fast-form-generator-react/src/styles/form-item.select.style.ts

```diff
@@ -0,0 +1,64 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
```

Comment:
Not a comment on this line, but I would not put these in their own folder, the can just be in src/form/ with the rest of the files

Reply:
ok